src/test/java/org/mule/extension/api/runtime/MutableContentMetadataTestCase.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.api.runtime;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.sameInstance;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+public class MutableContentMetadataTestCase extends AbstractContentMetadataTestCase
+{
+
+    @Override
+    protected ContentMetadata createContentMetadata()
+    {
+        return new MutableContentMetadata(inputContentType);
+    }
+
+    @Override
+    public void initialState()
+    {
+        outputContentType = inputContentType;
+        super.initialState();
+    }
+
+    @Test
+    public void isModifiable()
+    {
+        assertThat(contentMetadata.isOutputModifiable(), is(true));
+    }
+
+    @Test
+    public void setOutputContentType()
```

Comment:
This method can be on the super class and just override it on the test for the inmutable version